### DISCUSSION TITLE
Mention utilisateur dans le classement

### DISCUSSION THREAD

**Discussion by voltou91** [Crownicles/Crownicles#1750] [Nouvelles idées]
L'idée serait d'ajouter un champ optionnel où on peut mentionner un joueur sur la commande /classement (/top), le classement afficherait la page où se trouve le joueur en question.

**Comment by QQtin3** [comment]
Pas très bien compris, si tu fais ta commande /classement @ping ça te renverrai la page du classement où se trouve l'utilisateur ? Si c'est ça je vois pas très bien l'objectif

**Comment by QQtin3** [comment]
Fin si ça te renvoie la page oui, si ça te renvoie le numéro de page (ce que j'ai compris mais qui me paraît improbable) c'est un non pour ma part

**Reply by voltou91** [reply]
Non la page, ça affiche la page sur le classement, pas le numéro XD

**Comment by BastLast** [comment]
j'aime bien l'idée
